src/styles.css

```diff
@@ -100,6 +100,14 @@ ul {
   background-color: gainsboro;
 }
 
+#refresh {
+  font-size: 1rem;
+}
+
```

Comment:
- **[OPTIONAL]** If the user enters large content the app loses its UI flow like in the below screenshot:
![Screenshot from 2023-05-18 10-32-47](https://github.com/katfogy/leadership-board-game/assets/68030297/2b5a6a2b-1e96-4fc7-893b-b5fe700a5dd9)

- To fix this I highly recommend adding `word-break: break-all` to the CSS `score` as shown in this example: 

```css
.score {
    ...
    word-break: break-all;
    ...
}
```

## And here is the result:
![Screenshot from 2023-05-18 10-35-45](https://github.com/katfogy/leadership-board-game/assets/68030297/5881502f-3e6d-4041-895b-096be527f5ee)

The app layout remain consistent even with large content 👍🏼 💯